examples/codegen_verifier/codegen_verifier.sh

```diff
@@ -6,38 +6,46 @@ BACKEND=${BACKEND:-bb}
 nargo compile
```

Comment:
Why is this called `codegen_verifier`? and not `solidity_verifier`

Reply:
Because this was the original nargo command name `nargo codegen-verifier`